src/context/useAuth.utils.ts

```diff
@@ -237,14 +228,10 @@ export function getAddEthereumChainParameters(chainId: ChainId): AddEthereumChai
           symbol: 'MATIC',
           decimals: 18,
         },
-        rpcUrls: ['https://rpc-mumbai.maticvigil.com/'],
-        blockExplorerUrls: ['https://mumbai.polygonscan.com/'],
+        rpcUrls: ['https://rpc.decentraland.org/amoy'],
+        blockExplorerUrls: ['https://oklink.com/amoy'],
```

Comment:
I'm not sure this will work without an API key, but since it's for a testnet we can try

https://www.oklink.com/docs/en/#quickstart-guide-getting-started

<img width="777" alt="image" src="https://github.com/decentraland/governance-ui/assets/2858950/b2b73075-3b0a-49ca-b595-4de77d281b93">